uint16 logical_count
uint16 physical_count
uint64 ctx_switches
uint64 interrupts
uint64 soft_interrupts
uint64 syscalls
rcp/cpucore[] cpus